cli/input/input.go

```diff
@@ -48,16 +49,35 @@ func readLine(trm *term.Terminal, prompt string) (string, error) {
 
 // ReadPassword reads the user's password with prompt.
 func ReadPassword(prompt string) (string, error) {
-	trm := Terminal
-	if trm == nil {
-		s, err := term.MakeRaw(int(syscall.Stdin))
-		if err != nil {
-			return "", err
+	if runtime.GOOS == "windows" {
```

Comment:
I think we can decide on it during compilation.

Reply:
Should we create two separate files for UNIX and windows?

Reply:
Move one function to it, that's perfectly fine.